adoptium-updater-parent/adoptium-mappers-parent/adopt-mappers/src/main/kotlin/net/adoptium/api/v3/mapping/adopt/AdoptReleaseMapper.kt

```diff
@@ -219,7 +219,7 @@ private class AdoptReleaseMapper constructor(
     private suspend fun associateMetadataWithBinaries(releaseAssets: GHAssets): Map<GHAsset, GHMetaData> {
         return releaseAssets
             .assets
-            .filter { it.name.endsWith(".json") }
+            .filter { it.name.endsWith(".json") || ( it.name.contains("sbom") && it.name.endsWith("-metadata.json")) }
```

Comment:
I think this effectively has no change since `it.name.endsWith(".json")` would match `*-metadata.json` anyway

Reply:
you are absolutely right! made an update now